matrix/test_unittest_matrix.py

```diff
@@ -11,12 +11,12 @@
 
 
 class TestMatrixCreate(unittest.TestCase):
-    """Test the creation of a matrix and that an instance can't be modified."""
-
     def test_initialization(self):
         """Test init function of Matrix."""
-        self.assertIsInstance(Matrix(((1, 2), (3, 4))), Matrix)
-        self.assertIsInstance(Matrix(((1, 2), (5.5, 4))), Matrix)
+        matrix_data = (19, 48), (19, 67)
```

Comment:
when using tuples, either write:
matrix = ((1,2),(3,4))
or:
a, b = (1,2), (3,4)
i.e. if you want a single variable, make it clear that it represent a single variable